src/site/xdoc/manual/appenders.xml

```diff
@@ -1555,6 +1555,33 @@ public class ConnectionFactory {
               <th>Default</th>
               <th>Description</th>
             </tr>
+            <tr>
+              <td>allowdLdapClasses</td>
```

Comment:
I just noticed now that the documentation in `appenders.xml` has a typo in the property names. The docs say `allowd*` (missing `e`) while the actual code in [JndiManager](https://github.com/apache/logging-log4j2/pull/608/files#diff-271353c1076e53f6893261e4420de27d34588bfd782806b5c66a3465c43b7f51) looks up properties starting with `allowed`.